jquery.flot.js

```diff
@@ -1816,6 +1838,7 @@ Licensed under the MIT license.
             }
 
             surface.render();
+            webglsurface.render();
```

Comment:
Can this be moved in a ```draw``` hook? The draw is called a single times after "drawing" all the series, I expect this to work, but it has to be tested. Worst case we need to add one more hook called ```render``` which is called here.